From: Anthony Sottile <asottile@umich.edu>
Date: Thu, 21 Jun 2018 07:44:22 -0700
Subject: re-enable tab tearing

---
 src/terminal-notebook.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/terminal-notebook.c b/src/terminal-notebook.c
index 490a29a..0b43b7f 100644
--- a/src/terminal-notebook.c
+++ b/src/terminal-notebook.c
@@ -151,9 +151,7 @@ terminal_notebook_add_screen (TerminalMdiContainer *container,
                            "tab-fill", TRUE,
                            NULL);
   gtk_notebook_set_tab_reorderable (gtk_notebook, screen_container, TRUE);
-#if 0
   gtk_notebook_set_tab_detachable (gtk_notebook, screen_container, TRUE);
-#endif
 }
 
 static void
