From: Anthony Sottile <asottile@umich.edu>
Date: Wed, 1 Jun 2022 21:09:55 -0400
Subject: re-enable-tab-tearing

---
 src/terminal-notebook.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/terminal-notebook.c b/src/terminal-notebook.c
index 5f5670f..810841f 100644
--- a/src/terminal-notebook.c
+++ b/src/terminal-notebook.c
@@ -149,9 +149,7 @@ terminal_notebook_add_screen (TerminalMdiContainer *container,
                            "tab-fill", TRUE,
                            NULL);
   gtk_notebook_set_tab_reorderable (gtk_notebook, screen_container, TRUE);
-#if 0
   gtk_notebook_set_tab_detachable (gtk_notebook, screen_container, TRUE);
-#endif
 }
 
 static void
